src/components/promotion/PromotionImage.tsx

```diff
@@ -1,8 +1,22 @@
 import { Skeleton } from 'antd';
-import React, { ReactElement } from 'react';
+import React, { CSSProperties, ReactElement } from 'react';
 
 import { PromotionImage as PromotionImageProps } from '../../types/promotion';
 
-export default function PromotionImage({ src }: PromotionImageProps): ReactElement {
-  return <div>{src?.length ? <img src={src} alt="" /> : <Skeleton.Image />}</div>;
+const styles: { [identifier: string]: CSSProperties } = {
+  image: {
+    borderRadius: 15,
+    minHeight: '100%',
+    objectFit: 'cover',
+    width: 100,
+  },
+};
+
+export default function PromotionImage({
+  // TODO: remove default image
```

Comment:
Maybe we can link the issue to this comment as well? https://github.com/ubclaunchpad/PromoPal-backend/issues/101?

Reply:
Ahhh yes I'll add this thanks!